lib/chef_backup/strategy/restore/tar.rb

```diff
@@ -148,7 +148,11 @@ def reconfigure_server
 
   def cleanse_chef_server(agree)
     log 'Cleaning up any old files'
-    shell_out!("#{ctl_command} cleanse #{agree || ''}")
+    # The chef-server-ctl cleanse command deliberately waits 60 seconds to give
+    # you an option to cancel it.  Therefore, don't count it in the timeout that
+    # the user provided.
+    timeout = shell_timeout + 60 if shell_timeout
```

Comment:
I would need to double check but I think that if `agree` is `yes` then cleanse will skip the sleep.